_includes/footer.html

```diff
@@ -191,8 +181,8 @@ <h6 class="title">Instagram</h6>
 		<script async defer src="https://apis.google.com/js/platform.js" ></script>
 		<script src="js/contributors.js"></script>
 		<script src="js/inview.js"></script>
+		<script src="js/backtop.js"></script>
```

Comment:
This is for connect js that containing Back to top files with html file

Reply:
Why are you making new js file? You can simple append the code in `script.js`